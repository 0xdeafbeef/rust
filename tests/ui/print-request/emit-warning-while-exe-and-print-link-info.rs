//@ compile-flags: --print native-static-libs --crate-type staticlib  --emit metadata
//@ check-pass
//~? WARN skipping link step due to conflict: cannot output linkage information without emitting link
