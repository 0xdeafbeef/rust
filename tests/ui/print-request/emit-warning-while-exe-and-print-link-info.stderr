warning: skipping link step due to conflict: cannot output linkage information without emitting link

warning: 1 warning emitted

